PWR/PWR_button/Core/Src/main.c

```diff
@@ -1,80 +1,41 @@
 #include "stm32l053xx.h"
 
-#define BUTTON_DEBOUNCE	50
-
-uint8_t flag_EXTI 		= 0,
+//------ DEFINES
+//--- INPUTS
+#define USR_BT_PRESS		(!(GPIOA->IDR & GPIO_IDR_ID10_Msk))
+
+//--- OUTPUTS
+#define GREEN_LED_ON		(GPIOA->ODR |= GPIO_ODR_OD7_Msk)
+#define	GREEN_LED_OFF		(GPIOA->ODR &= ~GPIO_ODR_OD7_Msk)
+#define TOGGLE_GREEN_LED	(GPIOA->ODR ^=GPIO_IDR_ID7_Msk)
+
+//--- GENERAL
+#define BUTTON_DEBOUNCE	50		//constant for button debounce
+#define RELEASED		0		//user button status define
+#define	PRESSED			1		//user button status define
+
+uint8_t flag_EXTI 		= 0,	//this flag is set when there's an EXTI 10 interrupt
```

Comment:
we don't use it as an official convention yet but stick to the lowerCamelCase format when naming vars and funcitons. https://pt.wikipedia.org/wiki/CamelCase

Reply:
@rodrigocelmer saw your fix, but it would be the **lower**CamelCase version, meaning that the first letter is in lower case